tools/functional-tester/etcd-tester/cluster.go

```diff
@@ -109,6 +101,12 @@ func (c *cluster) bootstrap() error {
 		go c.Stressers[i].Stress()
 	}
 
+	if c.consistencyCheck && !c.v2Only {
+		c.Checker = newHashChecker(c)
```

Comment:
newHashChecker(hashAndRevGetter(c)) so you know that you do not give the hashchecker access to all methods on cluster.